### DISCUSSION TITLE
v1.1.0-alpha

### DISCUSSION THREAD

**Discussion by Butterroach** [slashsh/slash#5] [Announcements]
- Fix bug #3 
- Fix bug #4 
- Add `help` command
- Write README.md
- Add parameter to `$` specifying a specific variable name to show the contents of

<hr /><em>This discussion was created from the release <a href='https://github.com/slashsh/slash/releases/tag/1.1.0-alpha'>v1.1.0-alpha</a>.</em>

**Comment by Butterroach** [comment]
@OcaMinty idk why im mentioning you but FUCK you!!!!!!!

**Reply by OcaMinty** [reply]
penis
